client/flutter/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -203,7 +201,6 @@
 			files = (
 				97C147011CF9000F007C117D /* LaunchScreen.storyboard in Resources */,
 				3B3967161E833CAA004F5970 /* AppFrameworkInfo.plist in Resources */,
-				8B08AE422432233B00304E0E /* GoogleService-Info.plist in Resources */,
```

Comment:
do we want to remove GoogleService-Info.plist entirely? I think we still want the correct staging copy in there?

Reply:
Maybe I'm getting confused by something but the one in my project and the one in Master are the same "staging" google-services file. Are you suggesting we .gitignore it?

Reply:
sorry for not being clear with my earlier comment - this is what a lot of days of no sleep in a row will do to my ability to write 😅 

I meant that it looks like this change removes the GoogleServices-Info.plist file from this Xcode project. Even if it's in the file system in the repo, I'm not sure if Xcode would still need to hold the reference to it here in the `project.pbxproj` file. I'm assuming you would have seen the build break if so, but just making sure